mirascope/core/mistral/_call_kwargs.py

```diff
@@ -2,12 +2,17 @@
 
 from typing import Any
 
-from mistralai.models.chat_completion import ChatMessage
+from mistralai.models import (
+    AssistantMessage,
+    SystemMessage,
+    ToolMessage,
+    UserMessage,
+)
 
 from ..base import BaseCallKwargs
 from .call_params import MistralCallParams
 
 
 class MistralCallKwargs(MistralCallParams, BaseCallKwargs[dict[str, Any]]):
     model: str
-    messages: list[ChatMessage]
+    messages: list[AssistantMessage | SystemMessage | ToolMessage | UserMessage]
```

Comment:
nit: this should be typed as `Sequence`, and we should probably update any other providers as well